lib/facilities/ppms/v1/client.rb

```diff
@@ -159,7 +159,8 @@ def pos_locator_params(params, pos_code)
         end
 
         def provider_locator_params(params)
-          specialty_codes = params[:specialties].first(4).map.with_index.with_object({}) do |(code, index), hsh|
+          specialties = Array.wrap(params[:specialties])
+          specialty_codes = specialties.first(4).map.with_index.with_object({}) do |(code, index), hsh|
```

Comment:
I can confirm that your change fixes the specific issue of `undefined method `first' for nil:NilClass`. However, I am unable to determine if there might end up being additional errors further down stream (i.e. there may be a deeper root to this problem) so it may be worth keeping an eye on.

Reply:
Thanks, will keep an eye on it